spec/factories/organisations.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```

Comment:
Sorry for all these comments, but be happy because that means I can't find anything else :wink: magic comment?